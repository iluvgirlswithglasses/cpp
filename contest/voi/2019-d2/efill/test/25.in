38
-80057852 61516998
-80057826 -71433852
-68891372 -80047726
22016302 -80057874
-80045292 77947788
80024280 78723152
80042236 76273100
80058178 -34062604
80055072 -71685198
80052706 67159374
79917538 79558420
-65581902 -80052256
-78568980 -80028956
-80054352 76593310
79853802 79753650
80040204 77421870
79041632 -80000214
80044542 -78442550
65313302 -80056296
80058012 -65485190
-79380660 -79986602
79403104 -79978118
78089474 -80041482
35511336 80057582
6916656 80057754
-79992964 -79909538
79949992 -79926370
-66487744 80057816
-79937068 79809000
79594494 80045974
78949924 80056618
-38540872 -80056298
80055724 63255256
79765464 79938568
80057136 39384708
-80052858 -79020522
-7478064 -80057856
-79220804 80055130
52892998 61517024 52893024 -71433826
