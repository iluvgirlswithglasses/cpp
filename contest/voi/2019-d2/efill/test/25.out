796186657.03
